test/Api.Tests/testconfig.json

```diff
@@ -0,0 +1,6 @@
+﻿// http://docs.mytestedasp.net/tutorial/licensing.html#registering-a-license-code
+// Each license is per web application and hence this is according to our test assembly name.
+// Also it will for all the assemblies that starts with the name "Api".
+{
+  "License": "4433-a4DvjmN8PMZSeaJZs7pw/MQnBFLfu1XGG9PXem39ml5ObzGSdfww7joBm1ADPCxSqc61qK+Zk9dmm9k7vJMHxCe4ASG64LlxpfCkzqGz7XAE0nj/mqz5pTim3pf1OESUo+wGFbj9RvJhlln5kL7hNGz49beoA5ks4hJrLQzNrho0NDMzOjIwMTctMTItMDY6bmFnYXJhakBjYW1wdXMudW5pLXBhZGVyYm9ybi5kZTpVbml2ZXJzaXR5IG9mIFBhZGVyYm9ybiwgR2VybWFueTpBcHBsaWNhdGlvbjpBcGku"
```

Comment:
@nagaraj1993 As @sstamm already wrote multiple times, one shouldn't publish credentials here on GitHub. That's why we have configuration files which are not in the Git repository!